### DISCUSSION TITLE
It works and it doesn't work

### DISCUSSION THREAD

**Discussion by diegobkm** [nefarius/DsHidMini#274] [Support]
I performed all the procedures correctly and the program works, it recognized my original ds3 control (bluetooth and wired), but the only button that works is the home button. Through the dshm program I was testing the other modes, the only one that recognized all the commands was "sxs", but it only works in that specific steam mode, and only it works, not even on windows in general and no other program works. I wanted to use it on xbox cloud gaming, but I tried everything and I couldn't, I searched the threads and still couldn't find a solution.

**Comment by nefarius** [comment]
The solutions are documented a thousand times, you folks need to read and process them though. For modern games use DS4Windows (mode and app, as documented), the other modes have their specific use case, this is not a bug nor something to improve, it's intentional. Get used with the docs, it's written for a reason and covers your issues a 100%.
